library_generation/generate_showcase_integration_test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source $SCRIPT_DIR/utilities.sh
+SDK_WORKSPACE=$SCRIPT_DIR/../WORKSPACE
+GAX_PROPERTIES=$SCRIPT_DIR/../gax-java/dependencies.properties
+SHOWCASE_BUILD_FILE=$SCRIPT_DIR/../showcase/BUILD.bazel
+
+cd $SCRIPT_DIR
+rm -rdf output
+
+# clone gapic-showcase
+if [ ! -d schema ]; then
+  git clone https://github.com/googleapis/gapic-showcase
+  pushd gapic-showcase
+  showcase_version=$(get_version_from_WORKSPACE "_showcase_version" $SDK_WORKSPACE)
+  git checkout "v$showcase_version"
+  mv schema ..
+  popd
+  rm -rdf gapic-showcase
+fi
+if [ ! -d google ];then
+  git clone https://github.com/googleapis/googleapis
```

Comment:
We need some googleapis protos. Showcase protos [reference them](https://github.com/googleapis/gapic-showcase/blob/c6512469703a716c619058249f2bb55d81c5ee37/schema/google/showcase/v1beta1/BUILD.bazel#L33-L40).